tests/jerry/date-tostring.js

```diff
@@ -58,8 +58,8 @@ catch (e)
 
 assert (new Date (NaN).toTimeString () == "Invalid Date");
 assert (new Date (Number.POSITIVE_INFINITY).toString () === "Invalid Date");
-assert (new Date ("2015-02-13").toTimeString () == "00:00:00.000");
-assert (new Date ("2015-07-08T11:29:05.023").toTimeString () == "11:29:05.023");
```

Comment:
The change is good here, but could you please keep the originals and move these 4 asserts into `tests/jerry/es5.1/date-tostring.js`?

Reply:
The original asserts will fail with every profile because of the changed format, unless we keep the old implementation for the es5.1 and es6 builds. Only esnext specifies the correct format so I think it's unnecessary to have two different implementations. But of course I can do that if you prefer.